.codeclimate.yml

```diff
@@ -0,0 +1,47 @@
+version: "2"         # required to adjust maintainability checks
+checks:
+  argument-count:
+    config:
+      threshold: 4
+  complex-logic:
+    config:
+      threshold: 4
+  file-lines:
+    config:
+      threshold: 250
+  method-complexity:
+    config:
+      threshold: 5
+  method-count:
+    config:
+      threshold: 20
+  method-lines:
+    config:
+      threshold: 25
+  nested-control-flow:
+    config:
+      threshold: 4
+  return-statements:
+    config:
+      threshold: 4
+  similar-code:
+    config:
+      threshold: # language-specific defaults. an override will affect all languages.
+  identical-code:
+    config:
+      threshold: # language-specific defaults. an override will affect all languages.
+plugins:
+  rubocop:
+    enabled: true
+    channel: rubocop-0-52
+  brakeman:
```

Comment:
Does brakeman have a version/channel? We've got a warning on the newest version when I tried running it locally last week.

Reply:
nope. doesn't look like it supports a channel param https://docs.codeclimate.com/docs/brakeman